### DISCUSSION TITLE
Vertical align image within column

### DISCUSSION THREAD

**Discussion by ukserversltd** [bootstrap-email/bootstrap-email#159] [Help]
I am struggling to vertically align a image in a column, any ideas best way to do this, ive tried ay-text-bottom etc.. but not having any joys.

I have 2 columns, first one with 3 lines of text the second column a anchor button which i want vertically aligned to the bottom to it lines up.

`<div class="col-4">
              <div class="ax-right ay-text-bottom">
                  <a class="btn btn-white" href="#">BOOK NOW</a>
              </div>
          </div>`

**Comment by psociety** [comment]
When `text-center` class doesn't work, i just go for `style="margin-left: auto;margin-right: auto;"` as `mx-auto` doesn't seem to exist in that bootstrap version.

**Reply by ukserversltd** [reply]
That will place it in the centre which is all good, i need to vertically align the element to the bottom

**Reply by stuyam** [reply]
@psociety `ax-center` is the equivalent of `mx-auto` in bootstrap and has better email support.

**Comment by stuyam** [comment]
This worked for me, you want you `ax` and `ay`'s on you col. Just tried it in the online editor:
```html
<div class="row">
  <div class="col-6"> 
    Three<br>
    Lines<br>
    of Text
  </div>
  <div class="col-6 ax-right ay-bottom"> 
    <a class="btn btn-white" href="#">BOOK NOW</a> 
  </div>
</div>
```

**Reply by ukserversltd** [reply]
That it does, thanks mate!
